code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -475,6 +477,20 @@
 
 				return
 
+		if(istype(W, /obj/item/weapon/implant/restrainingbolt) && !cell)
+			if(bolt)
+				to_chat(user, "<span class='notice'>There is already a restraining bolt installed in this cyborg.</span>")

```

Comment:
So if a borg breaks their bolt, they can't be re-bolted until someone with access actually takes out the defective bolt?

Reply:
It takes access to place the bolt _in_ the borg, as the panel needs to be open at the moment.